modules/nodes-containers-sysctls-about.adoc

```diff
@@ -6,94 +6,17 @@
 [id="nodes-containers-sysctls-about_{context}"]
 = About sysctls
 
-In Linux, the sysctl interface allows an administrator to modify kernel
-parameters at runtime. Parameters are available via the *_/proc/sys/_* virtual
-process file system. The parameters cover various subsystems, such as:
+In Linux, the sysctl interface allows an administrator to modify kernel parameters at runtime. Parameters are available from the `_/proc/sys/_` virtual process file system. The parameters cover various subsystems, such as:
```

Comment:
I see you changed the bold to monospace for these items, but technically they shouldn't be italics either. Not critical to merge this PR though, so I'd ask that either you remove the italics in this PR, or make a note to follow up in a separate PR to remove them throughout.

Reply:
Sounds good, thanks!